assets/js/app.js

```diff
@@ -31,10 +31,10 @@ const hooks = {
       const { inputSelector } = this.el.dataset;
       this.el.addEventListener("click", (ev) => {
         ev.preventDefault();
-        const $targetInput = document.querySelector(inputSelector);
-        $targetInput.select();
-        $targetInput.setSelectionRange(0, 99999);
-        navigator.clipboard.writeText($targetInput.value);
+        const targetInput = document.querySelector(inputSelector);
```

Comment:
That's not the reason, dollar sign leading a variable name is a convention to say this variable is a node on DOM.

Reply:
Got it. This is a pretty odd issue given that this is what the documentation suggests, what you already had, for both firefox and chrome. I have no experience with Elixir, so I must ask, when you said open a room in firefox, how would I go about doing that? Maybe I could troubleshoot that better.

Reply:
![image](https://user-images.githubusercontent.com/9121359/196266202-57c050ca-f2df-423f-a9d4-9ea8aa1fc78c.png)

go to the main page and create a room

Reply:
Hey Lubien, I just tried the room and clicked on the button, and it copied the link just fine on Firefox desktop and android as well. Could it be that it's just not working for you on your end?

Reply:
Weird, it did not seem to work a couple days ago. I'll add this to the issue for now

Reply:
cool, glad I was able to trouble shoot this, and that it now works.